### ISSUE TITLE
[Admin] "Users" doesn't work

### ISSUE THREAD

**Issue by clickwork-git** [bludit/bludit#1019]
On smartphones users
* cannot edit their profiles,
* can delete or edit posts of other users.

**Comment by erdifr** [comment]
I also experienced this on desktop. Users (role = editor) could not edit themselves because they are not granted access in `users.php`. Changing line 7 of `bludit/bl-kernel/admin/controllers/users.php` from:
```php
checkRole(array('admin'));
```
to:
```php
checkRole(array('admin', 'editor'));
```
allowed users with role of `editor` to access `users.php` and in turn to edit themselves. I believe that this would fix part 1 of #1019. This does **not** address part 2 of #1019 in any way.

**Example Part 1:** After changing `users.php` as stated above user `Editor` with `role = editor` was able to access `users.php`. From `users.php` `Editor` was able to click on _**any**_ listed username and edit `user` `Editor`. `user` `Editor` was **not** able to edit any other `user`.

**Example Part 2:** Any `user` with `role = editor` **can delete** content from **all** users (including `role = admin`). Maybe this was intended behavior? I do not know.

I do see a case for restricting deletion of `content` to the `user` that created the `content`. Besides the `user` that created the `content` I believe that `admin` users should be able to delete any `content` regardless of whether or not they authored it.

**Comment by dignajar** [comment]
I'm thinking to no create a new role and use the current role `Editor` to only allow him to edit his/her own content, make sense ? or is better to keep `Editor` and create a new role like `Writer` ?

**Comment by BlakesHeaven** [comment]
I think `Editor` and `Writer` are much the same without really saying what the difference is. 
I'd  go with `Sole Editor` and `Full Editor`.

I also suggest total separation between Admin functionally and Content functionality, and add in option for multiple
Roles to combine both if both are needed.
E.g., I administer a site but do not edit anything, so don't need edit access.

**Comment by dignajar** [comment]
@clickwork-git let me know if this issue is fixed in Bludit v3.9.0 beta 1.

**Comment by dignajar** [comment]
Hi,
I close the issue, please take a look to the version 3.9 from Github with the new features, if you still have a problem feel free to open the issue.
Regards
